src/plugins/kibana/server/lib/case_conversion.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash');
+
+module.exports = {
+  convertToSnakeCase: function (object) {
+    return _.mapKeys(object, (value, key) => {
+      return _.snakeCase(key);
+    });
+  },
+
+  convertToCamelCase: function (object) {
```

Comment:
In some cases that's the case, `_.merge` is an exception, and in general documentation never hurts.